morpheus/loaders/file_to_df_loader.py

```diff
@@ -88,7 +88,8 @@ def file_to_df_loader(control_message: ControlMessage, task: dict):
     parser_kwargs = config.get("parser_kwargs", None)
     cache_dir = config.get("cache_dir", None)
 
-    downloader = Downloader()
+    if not hasattr(file_to_df_loader, "downloader"):
+        file_to_df_loader.downloader = Downloader()
```

Comment:
I'm confused. Who is calling `file_to_df_loader` and why are they not responsible for making sure there is only one instance?